tests/models/roformer/test_modeling_roformer.py

```diff
@@ -513,8 +513,9 @@ class RoFormerSinusoidalPositionalEmbeddingTest(unittest.TestCase):
 
     def test_basic(self):
         input_ids = torch.tensor([[4, 10]], dtype=torch.long, device=torch_device)
-        emb1 = RoFormerSinusoidalPositionalEmbedding(num_positions=6, embedding_dim=6).to(torch_device)
+        emb1 = RoFormerSinusoidalPositionalEmbedding(num_positions=6, embedding_dim=6)
         emb1._init_weight()
+        emb1 = emb1.to(torch_device)
```

Comment:
For reviewers:

`emb1._init_weight()` called above creates tensors and nn modules which are not getting moved to the GPU device as `.to(device)` was called before that and never repeated. See:
https://github.com/huggingface/transformers/blob/78d771c3c21922642fc9546ccb973cc7a182ab34/src/transformers/models/roformer/modeling_roformer.py#L54-L65